homeassistant/components/smarttub/config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Config flow to configure the SmartTub integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, CONF_SCAN_INTERVAL
+from homeassistant.core import callback
+
+from .const import DEFAULT_SCAN_INTERVAL, DOMAIN  # pylint: disable=unused-import
+from .controller import SmartTubController
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_EMAIL): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SmartTubConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """SmartTub configuration flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler(config_entry)
+
+    async def async_step_import(self, import_info):
+        """Set the config entry up from yaml."""
+        return await self.async_step_user(import_info)
```

Comment:
This isn't covered by tests, but should be removed anyway. Nothing to import since this is a new integration